docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
+  api:
+    image: build2024/app:latest
+    ports:
+      - "9000:9000"
+    environment:
+      - FRONT_URL
```

Comment:
Maybe you misunderstood these environment variable settings. Here, what is expected is specifying like below 
https://github.com/Ayana326/mercari-build-training/blob/005a4638bfed9fab25b4f2ce0549a61680b37109/composetest/docker-compose.yml#L10

What I mean here is not syntax difference (map vs array) but defining key: value (or key=value) pair.

fyi, map / array syntax is describing in https://docs.docker.com/compose/compose-file/05-services/#environment


Note: In addition, maybe the reason why you didn't encounter CORS is because localhost:3000 is specified (this is because if you don't specify value of environment it is unset and default value `http://localhost:3000` is set).
https://github.com/Ayana326/mercari-build-training/blob/1548f5a2cd7432c44056883b191d133d3baf1e39/python/main.py#L17